### PR TITLE
Updated no-db output, added module commands

### PR DESCRIPTION
**Pull request by Articdive**

### CODE REVIEW

File: src/main/java/com/dreamhorizon/core/commands/ModuleCommand.java [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+@Description("Module administration command")
+@CommandPermission("dhcore.command.modules")
+public class ModuleCommand extends DHCommand {
+    
+    @Subcommand("list")
+    @Description("Gets a list of enabled modules")
+    @Syntax("(page)")
+    @CommandPermission("dhcore.modules.list")
+    public static void onModuleList(CommandSender sender, @Default(value = "1") int page) {
+        if (page == 0) page = 1;
```
Comment (CollinHerber) [inline src/main/java/com/dreamhorizon/core/commands/ModuleCommand.java]: Wrap curlys around this. I won't stand for no curlys

File: src/main/java/com/dreamhorizon/core/commands/ModuleCommand.java [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+        List<ModuleEntry> paginatedModules = PaginationUtil.getPage(ModuleHandler.getInstance().getModuleEntries(), page, 8);
+        int maxPage = (int) Math.ceil(paginatedModules.size() / 8.0);
+        if (maxPage == 0) maxPage = 1;
```
Comment (CollinHerber) [inline src/main/java/com/dreamhorizon/core/commands/ModuleCommand.java]: Wrap curlys around this. I won't stand for no curlys

File: src/main/java/com/dreamhorizon/core/commands/ModuleCommand.java [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+    @Syntax("[module]")
+    @CommandCompletion("@modules")
+    @CommandPermission("dhcore.modules.info")
+    public static void onModuleInfo(CommandSender sender, ModuleEntry moduleEntry) {
+        HashMap<String, Object> placeholders = new HashMap<>();
+        placeholders.put("moduleName", moduleEntry.getName());
+        placeholders.put("moduleAuthor", moduleEntry.getAuthor());
+        placeholders.put("moduleCommandAmount", moduleEntry.getModule().getCommands().size());
+        placeholders.put("moduleListenerAmount", moduleEntry.getModule().getListeners().size());
+        if (moduleEntry.getModule().isEnabled()) {
```
Comment (CollinHerber) [inline src/main/java/com/dreamhorizon/core/commands/ModuleCommand.java:95]: Perhaps a ternary operator here? Not needed, just a thought.

**Review by CollinHerber — CHANGES_REQUESTED** [review]

File: src/main/java/com/dreamhorizon/core/commands/ModuleCommand.java (hunk 3 above)
Comment (Articdive) [reply src/main/java/com/dreamhorizon/core/commands/ModuleCommand.java:95]: I hate ternary operations. just a heads up :>

**Pull request merged**